### PR TITLE
Make classes more discoverable & update insert example urls button

### LINKED ISSUE #378: Make class creation more discoverable
**Tasks**
In order to make the class creation more discoverable, update the text `Search...` by `Search or Add...`. Icon is also updated
- [x] Update the class menu on the labelling interface
- [x] Update the context class menu (menu which appears after a right click on a label)
- [x] [Mockup](https://www.figma.com/file/4J9xyfF5v4B98MuKVO0Y8V/Labelflow-mockups?node-id=697%3A453)
image

### PR DESCRIPTION
**Pull request by GuilhemdV**
# Feature

## Work performed

- Updated the class-selection-list component default input value to `Search or Add...`.
- Cherry picked changes Vincent made in the utils to detect the app is running in dev mode.
- Updated the insert-example-urls button to `pink.200` with a dev icon to the left of the text, and made the button hide if not running in dev mode.

## Resolved issues

Closes #378 & #379

### CODE REVIEW

**Comment by nicOwlas** [comment]
- ✅ for the `Search or Add...` text
- ✅ for demo images button which is removed in prod
- ❓how can I test the demo image button available in dev mode?

File: typescript/web/src/components/import-button/import-images-modal/modal-url-list/url-list.tsx [hunk 1]
```diff
@@ -19,9 +23,16 @@ export const UrlList = ({
...
-      </Button>
+      {isDevelopmentEnvironment && (
+        <Button
+          mt={2}
+          onClick={() => setValue(imageSampleCollection.join("\n"))}
+          background="pink.200"
+        >
+          <DevIcon size="1.5rem" mr="1" />
+          Insert example images
+        </Button>
```
Comment (RaynalHugo) [inline typescript/web/src/components/import-button/import-images-modal/modal-url-list/url-list.tsx:34]: Nitpicking: 

I think a more standard way would be to do: 
```suggestion
        <Button
          mt={2}
          leftIcon={<DevIcon size="1.5rem" />}
          onClick={() => setValue(imageSampleCollection.join("\n"))}
          background="pink.200"
        >
          Insert example images
        </Button>
```

**Review by RaynalHugo — COMMENTED** [review]
I think the CI is failing because one Cypress test was depending on the button you removed. I guess you can just copy the import part of another test.

File: typescript/web/src/components/import-button/import-images-modal/modal-url-list/url-list.tsx (hunk 1 above)
Comment (GuilhemdV) [reply typescript/web/src/components/import-button/import-images-modal/modal-url-list/url-list.tsx]: Indeed 🤔 I tried this at first but it yielded very bad looking result so I went with the later. Guess the `size` part was off

**Comment by GuilhemdV** [comment]
>❓how can I test the demo image button available in dev mode?

You have to run the app in dev mode I guess ¯\\\_(ツ)\_/¯

**Comment by crubier** [comment]
> > ❓how can I test the demo image button available in dev mode?
> 
> You have to run the app in dev mode I guess ¯\_(ツ)_/¯

law-of-attraction-brain

**Comment by GuilhemdV** [comment]
image

**Review by crubier — APPROVED** [review]

**Comment by nicOwlas** [comment]
Lol the question was more: is it enough to run the app locally to be in dev mode?

**Comment by crubier** [comment]
`yarn dev:web` locally and yes you're in dev mode

**Comment by nicOwlas** [comment]
Ok thanks. Works well 👍

**Pull request merged**